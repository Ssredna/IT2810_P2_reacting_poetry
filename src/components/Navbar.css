.Navbar-container {
  height: 3.5rem;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.Navbar-title {
  font-size: 1.3rem;
  color: white;
}

.Navbar-navlink {
  margin-left: 1rem;
  text-decoration: none;
  color: white;
  font-size: 1rem;
}
@media screen and (min-width: 37.5em) {
  .Navbar-title {
    font-size: 1.6rem;
  }
}

@media screen and (min-width: 75em) {
  .Navbar-title {
    font-size: 1.9rem;
  }
}
