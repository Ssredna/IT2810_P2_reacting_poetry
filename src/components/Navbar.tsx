import React from 'react';
import {Link} from 'react-router-dom';
import './Navbar.css';
import '../ColorPalette.css';
import '../Container.css';

const Navbar = () => {
  return (
    <div className="bg-primary-3 Navbar-container General-container">
      <div>
        <span className="Navbar-title">Navbar</span>
      </div>
      <div>
        <Link className="Navbar-navlink" to="/gallery">
          Gallery
        </Link>
        <Link className="Navbar-navlink" to="/">
          Home
        </Link>
      </div>
    </div>
  );
};

export default Navbar;
