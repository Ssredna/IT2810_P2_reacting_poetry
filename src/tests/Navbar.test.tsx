import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

test('it renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
