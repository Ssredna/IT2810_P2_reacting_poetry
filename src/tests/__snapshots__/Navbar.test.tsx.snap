// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="bg-primary-3 Navbar-container General-container"
>
  <div>
    <span
      className="Navbar-title"
    >
      Reacting poetries
    </span>
  </div>
  <div>
    <a
      className="Navbar-navlink"
      href="/gallery"
      onClick={[Function]}
    >
      Gallery
    </a>
    <a
      className="Navbar-navlink"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </div>
</div>
`;
